osquery/logger/plugins/tests/buffered_tests.cpp

```diff
@@ -215,19 +215,19 @@ TEST_F(BufferedLogForwarderTests, test_async) {
   EXPECT_CALL(*runner, send(ElementsAre("foo"), "result"))
       .WillOnce(Return(Status(0)));
   runner->logString("foo");
-  std::this_thread::sleep_for(1.5 * kLogPeriod);
+  std::this_thread::sleep_for(5 * kLogPeriod);
```

Comment:
Can you issue all of the `EXPECT_CALL`s then have a single `this_thread::sleep_for` at the end of the test case?